### DISCUSSION TITLE
Combined "Following" and "For you" mode.

### DISCUSSION THREAD

**Discussion by vi** [community/community#37156] [Other Feature Feedback, Questions, & Ideas]
Can there be a third tab "Combined", which includes both for "Following" and "For you" content as a single timeline?
Entries should be discernible by style e.g. background colour.

**Comment by vi** [comment]
Obviously, I didn't mean to outright abolish the "Following" tab.

**Reply by Codel1417** [reply]
so its your fault? /s
